modules/page/html/render/profile.js

```diff
@@ -142,7 +142,11 @@ exports.create = function (api) {
     ])
 
     var prepend = h('header', { className: 'ProfileHeader' }, [
-      h('div.image', api.about.html.image(id)),
+      h('div.image', {
+          classList: [when(contact.hidden, '-hidden-profile-image', null)]
```

Comment:
Not a huge deal, but the (unwritten) convention for class names in mcss is to use camelCase e.g. `-hiddenProfileImage`

Multi-word classes aren't that common, but I found some examples here: [styles/base/profile-header.mcss](https://github.com/ssbc/patchwork/blob/master/styles/base/profile-header.mcss#L29)